ci/jenkins/test.sh

```diff
@@ -703,8 +703,10 @@ function run_conformance {
     kubectl rollout status daemonset/antrea-agent -n kube-system
 
     set +e
-    if [[ "$TESTCASE" =~ "conformance" ]]; then
+    if [[ "$TESTCASE" =~ "conformance" || "$TESTCASE" == "all-feature-conformance" ]]; then
```

Comment:
The all-feature-conformance and conformance tests use different test suites. Please refer to: https://github.com/antrea-io/antrea/blob/92454fb8e9fc614808b96fe14f120a69dc41bb8e/ci/jenkins/test-vmc.sh#L620 and https://github.com/antrea-io/antrea/blob/92454fb8e9fc614808b96fe14f120a69dc41bb8e/ci/jenkins/test-vmc.sh#L622.

Reply:
Also, please check https://github.com/antrea-io/antrea/blob/92454fb8e9fc614808b96fe14f120a69dc41bb8e/ci/jenkins/test-vmc.sh#L595 about yaml changes for all-feature conformance test.